test/npb/ft/npadmana/ft_transposed_comp_only.chpl

```diff
@@ -0,0 +1 @@
+ft_transposed.chpl
```

Comment:
Is this saying that this file literally contains this string, or is this how GitHub shows symbolic links?  (I'm wondering if I've ever reviewed a PR with symbolic links before).

Reply:
This is how github displays symlink